providers/base/bin/acpi_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OEM_OSI_LIST=( Linux-Dell-Video Linux-Lenovo-NV-HDMI-Audio Linux-HPI-Hybrid-Graphics )
+for osi in "${OEM_OSI_LIST[@]}"
+do
+	grep -q -r "$osi" /sys/firmware/acpi/* && exit 1
```

Comment:
I have one more suggestion to make (it's a one liner though!):

Currently, if there is an error, the script simply exits with exit code 1.

Could you `printf/echo` one sentence to explain why the script fails? That would help the review of the test reports.